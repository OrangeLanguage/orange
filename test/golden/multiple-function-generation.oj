def const(x, y) x
def tsnoc(x, y) y