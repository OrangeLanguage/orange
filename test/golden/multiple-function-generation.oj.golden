(() => {
  const const = (function (_handle, x, y) {
    x = x();
    y = y();
    return x;
  });
  const tsnoc = (function (_handle, x, y) {
    x = x();
    y = y();
    return y;
  });
  const;
  return tsnoc;
})()